doc/source/conf.py

```diff
@@ -15,12 +15,7 @@
 
 import datetime
 import subprocess
-from collections import OrderedDict
-
 import MDAnalysis as mda
```

Comment:
```suggestion

import MDAnalysis as mda
```
🤣 sorry I can't help myself from this